### DISCUSSION TITLE
missing (not available) abook > ebook

### DISCUSSION THREAD

**Discussion by skyzuma** [advplyr/audiobookshelf#780] [Ideas]
hello,

idk if this possible and how this can be realised but i have an ebook server (calibre / calibre-web) - with same content and missing books. Its possible to link to the ebook server if an book is missing or not available as an audiobook? Maybe as an link field in the edit window or as an icon beside the other icons (edit / marks a finished / ...)

**Comment by advplyr** [comment]
I don't see how this could be implemented that would make sense

**Comment by skyzuma** [comment]
the problem is: the experimental feature eBook: we need the *.epub AND the extracted files from the *.epub on the same folder (maybe this can fix in the future) ... so i use an external ebook server ... 

for internal eBook service:

if an audiobook is missing, the missing book can be set as an fake aBook. the img from abook / ebook and linked to the internal eBook library item ... if i click on the cover / link it goes to the eBook library item ... or its skinned as an eBook (with the litte book read sign)

for external use:

maybe the same as internal but the link goes to the external service ... 

i see, its difficult ...
